apps/api/src/server/app/validation/validation.controller.js

```diff
@@ -15,6 +15,25 @@ const appealsList = [
 	}
 ];
 
-export const getValidation = function (request, response) {
+const appealReview = [
+	{
+		AppealId : 1,
+		AppealReference: 'APP/Q9999/D/21/1345264',
+		AppellantName: 'Lee Thornton',
+		AppealStatus:'new',
+		Received: '23 Feb 2022',
+		AppealSite:'96 The Avenue, Maidstone, Kent, MD21 5XY',
+		LocalPlanningDepartment: 'Maindstone Borough Council',
+		PlanningApplicationReference: '48269/APP/2021/1482'
+	}
+];
```

Comment:
Just one last point, I don't believe we need to return an array here. I think we will only ever need to return a hash, so we can remove those square braces.